com.unity.render-pipelines.high-definition/Runtime/Lighting/VolumetricLighting/VolumetricLighting.compute

```diff
@@ -779,17 +678,15 @@ void VolumetricLighting(uint3 dispatchThreadId : SV_DispatchThreadID,
 
 #ifdef VL_PRESET_OPTIMAL
     // The entire thread group is within the same light tile.
-    uint2 tileCoord = groupOffset * VBUFFER_VOXEL_SIZE / TILE_SIZE_BIG_TILE;
+    uint2 tileCoord = groupOffset * VBUFFER_VOXEL_SIZE / TILE_SIZE;
+    uint  tile      = IndexFromCoordinate(tileCoord, TILE_BUFFER_DIMS.x);
 #else
     // No compile-time optimizations, no scalarization.
-    uint2 tileCoord = pixelCoord / TILE_SIZE_BIG_TILE;
+    uint  tile      = ComputeTileIndex(pixelCoord);
 #endif
-    uint  tileIndex = tileCoord.x + _NumTileBigTileX * tileCoord.y;
-    // This clamp is important as _VBufferVoxelSize can have float value which can cause en overflow (Crash on Vulkan and Metal)
```

Comment:
We need to keep this clamp in a way or the other, of do similar clamping, we have spend several QA and programmer hour on this one :)